### PR TITLE
docs:rpc-go updates and maintenance command

### PR DESCRIPTION
**Pull request by Kimberly-Davis**
## PR Checklist

- [ ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

docs/GetStarted/buildRPC.md
 .\docs\Reference\RPC\commandsRPC.md
mkdocs.yml
## Anything the reviewer should know when reviewing this PR?

**Yes. Plenty!**

I was not able to test the Linux commands. My linux system is non-operational at the moment, and the OS has to be re-installed.
@rsdmike Can you take a look at the Docker (On Linux Host Only) in buildRPC.md. I don't know what I'm doing there.

@matt-primrose @rsdmike 
Can you take a look at the tables in the amtinfo section of the commandsRPC.md. BW mentioned that you might have some interesting insights into what might go in the description section of these tables. 

I had a hard time deciding if I should do this or not, but I wanted to try exhaustively documenting this application. What does that look like? Can we keep the two things in sync?

@bwendlandt-intel @madhavilosetty-intel 
I think any insight you have will be greatly appreciated.

### CODE REVIEW

File: docs/Reference/RPC/commandsRPC.md [hunk 1]
```diff
@@ -1,100 +1,171 @@
...
+    sudo ./rpc deactivate -u wss://server/activate
+    ```
+=== "Windows"
+    ```
+    rpc deactivate -u wss://server/activate
+    ```
+
+| OPTION&emsp;&emsp;&emsp;&emsp;&emsp;&emsp; | DESCRIPTION |
+| -------------------------- | ---------------------- | 
+| -f | DNS suffix override |
```
Comment (matt-primrose) [inline docs/Reference/RPC/commandsRPC.md:79]: this should be -d

File: docs/GetStarted/buildRPC.md [hunk 2]
```diff
@@ -12,66 +20,101 @@ After running the RPC, the Intel® AMT managed device can be managed remotely us
...
 
+2. Confirm a successful build:
 
+    === "Linux"
+        ``` bash
+        sudo rpc version
+        ```
+    === "Docker (On Linux Host Only)"
+        ``` bash
+        placeholder for Mike
```
Comment (rsdmike) [inline docs/GetStarted/buildRPC.md:107]:
```suggestion 
sudo docker run --device=/dev/mei0 rpc-go:latest version
```

File: docs/GetStarted/buildRPC.md [hunk 3]
```diff
@@ -12,66 +20,101 @@ After running the RPC, the Intel® AMT managed device can be managed remotely us
...
+    === "Linux"
+        ``` bash
+        sudo apt install build-essential
+        ```
+        ``` bash
+        go build -o rpc ./cmd
+        ```
+    === "Docker (On Linux Host Only)"
+        ``` bash
+        placeholder for Mike
```
Comment (rsdmike) [inline docs/GetStarted/buildRPC.md:89]:
```suggestion 
docker build -f "Dockerfile" -t rpc-go:latest .
```

**Review by rsdmike — DISMISSED** [review]

File: docs/GetStarted/buildRPC.md [hunk 4]
```diff
@@ -12,66 +20,101 @@ After running the RPC, the Intel® AMT managed device can be managed remotely us
...
 
-11. Once the download is complete, click the completed job which will feature a green checkmark.
+## Get the RPC
 
-12. Download the appropriate RPC for your managed device's OS under the **Artifacts** section.
+**To clone the repository:**
 
+1. Open a Terminal or Command Prompt and navigate to a directory of your choice for development:
+   ``` bash
+   git clone --recursive https://github.com/open-amt-cloud-toolkit/rpc-go
```
Comment (bwendlandt-intel) [inline docs/GetStarted/buildRPC.md:66]: Don't need --recursive

File: docs/GetStarted/buildRPC.md [hunk 5]
```diff
@@ -12,66 +20,101 @@ After running the RPC, the Intel® AMT managed device can be managed remotely us
...
-docker build -f "Dockerfile" -t rpc:latest .
-```  
+    === "Linux"
+        ``` bash
+        sudo apt install build-essential
+        ```
+        ``` bash
+        go build -o rpc ./cmd
+        ```
+    === "Docker (On Linux Host Only)"
```
Comment (bwendlandt-intel) [inline docs/GetStarted/buildRPC.md:87]: Can we move Docker to after Windows for these tabbed sections? Linx+Windows is our preferred GSG path

File: docs/GetStarted/buildRPC.md [hunk 6]
```diff
@@ -12,66 +20,101 @@ After running the RPC, the Intel® AMT managed device can be managed remotely us
...
 
-!!! note
-    If you wish to perform this docker build on Windows, ensure **Use the WSL 2 based engine** is enabled in Docker Desktop.
-    Once the build is complete, you will need to get the built image to your Linux system running Docker.
 
+2. Confirm a successful build:
 
+    === "Linux"
+        ``` bash
+        sudo rpc version
```
Comment (bwendlandt-intel) [inline docs/GetStarted/buildRPC.md:103]: Double check all the Linux cmds, they need to be 'sudo ./rpc cmd-here', ./ is requried

**Review by bwendlandt-intel — CHANGES_REQUESTED** [review]

**Review by matt-primrose — APPROVED** [review]

**Review by bwendlandt-intel — APPROVED** [review]

**Pull request merged**